packages/theme/src/theme/_navs.scss

```diff
@@ -113,6 +97,39 @@
 					background: rgba(0, 0, 0, 0.03);
 				}
 			}
+
+			&.active {
+				> a,
+				> button {
+					box-shadow: 0 -2px 0 $brand-primary inset;
+
+					&,
+					&:focus,
+					&:hover {
+						background: transparent;
+						border: none;
+					}
+				}
+			}
+
+			&.disabled {
+				> a,
+				> button {
+					cursor: not-allowed;
+					color: rgba(1, 1, 1, 0.4);
+
+					&,
+					&:focus,
+					&:hover {
+						background: transparent;
+						border: none;
+					}
+				}
+			}
+		}
+
+		+ .tab-content {
+			padding-top: $padding-larger;
```

Comment:
No it's because of Bootswatch example embedded style. Let me fix that. :ok_hand: